test/suites/smoke-test-common/test-inflation-rewards.ts

```diff
@@ -93,8 +93,9 @@ describeSuite({
 
                 const supplyAfter = (await apiAtIssuanceAfter.query.balances.totalIssuance()).toBigInt();
 
-                // expected issuance block increment in prod is: 19n/1_000_000_000n
-                const expectedIssuanceIncrement = (supplyBefore * 19n) / 1_000_000_000n;
+                // expected issuance block increment in prod
+                const expectedIssuanceIncrement =
+                    runtimeVersion > 500 ? (supplyBefore * 9n) / 1_000_000_000n : (supplyBefore * 19n) / 1_000_000_000n;
```

Comment:
This could have `const inflationRate = ...` as a separate variable, to make it easier to change again in the future.
```suggestion
                    (supplyBefore * inflationRate) / 1_000_000_000n;
```

Reply:
Yes. For this I tried using `PerBill`, but problem is that it returns `BN` instead of `bigint` as result of the operation and I tried to find a way to convert (possibly I do not know enough typescript to do this). 

So, I left it as it is.

Reply:
Ok it's fine